CDCgov/prime-reportstream
# feat: pin node version

pins node version at 14.20.0 for local and ci builds, and adjusts readmes to include information on usage of nvm

See:
* https://github.com/actions/setup-node/blob/main/docs/advanced-usage.md#node-version-file

## Test Steps:
1. look at latest frontend build for this branch
2. _VERIFY_: build ran on version 14.20 and passed as expected
3. also ensure that building and testing on your local machine works as expected, following directions from readmes as necessary to use nvm to install version 14.20

<img width="1022" alt="Screen Shot 2022-09-29 at 9 57 02 AM" src="https://user-images.githubusercontent.com/92806979/193051661-844a214f-6f10-4015-9289-96e526c9830f.png">


## Changes
- *Include a comprehensive list of changes in this PR*
- *(For web UI changes) Include screenshots/video of changes*

## Checklist

### Testing

### Process

## Linked Issues
- Fixes #issue

## To Be Done
*Create GitHub issues to track the work remaining, if any*
- #issue

## Specific Security-related subjects a reviewer should pay specific attention to
- Does this PR introduce new endpoints?
    - new endpoint A
    - new endpoint B
- Does this PR include changes in authentication and/or authorization of existing endpoints?
- Does this change introduce new dependencies that need vetting?
- Does this change require changes to our infrastructure?
- Does logging contain sensitive data?
- Does this PR include or remove any sensitive information itself?

If you answered '_yes_' to any of the questions above, conduct a detailed Review that addresses at least:

- What are the potential security threats and mitigations? Please list the _STRIDE_ threats and how they are mitigated
    - **S**poofing (faking authenticity)
        - Threat _T_, which could be achieved by _A_, is mitigated by _M_
    - **T**ampering (influence or sabotage the integrity of information, data, or system)
    - **R**epudiation (the ability to dispute the origin or originator of an action)
    - **I**nformation disclosure (data made available to entities who should not have it)
    - **D**enial of service (make a resource unavailable)
    - **E**levation of Privilege (reduce restrictions that apply or gain privileges one should not have)
- Have you ensured logging does not contain sensitive data?
- Have you received any additional approvals needed for this change?
